app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-permission android:name="android.permission.INTERNET" />
+
```

Comment:
po co to jest?

Reply:
ok, bez tegho nie chce wyswietlac filmu :D

Reply:
ale dziwne jest to ze trzeba miec dostep do neta zeby z lokala ogladac filmik, moze to jakos inaczej powinno byc zrobiione